src/ddsrt/include/dds/ddsrt/heap.h

```diff
@@ -26,6 +26,21 @@
 extern "C" {
 #endif
 
+typedef struct ddsrt_allocation_ops {
+  void* (*malloc)  (size_t);
```

Comment:
It is a tantamount to nitpicking, but I'd prefer to see names for the arguments here. They're meaningless to the compiler and any C programmer may reasonably be assumed to know the what the arguments of these functions are, but it is cheap documentation and sometimes comes in very handy.

Reply:
No problem, I will add these!